Stop building the tar.gz distribution.

From: Lars Francke <git@lars-francke.de>

All we do is build Druid tar and gzip it only to immediately uncompress it
again. So, instead we just skip the compression step entirely.
---
 distribution/src/assembly/assembly.xml |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/distribution/src/assembly/assembly.xml b/distribution/src/assembly/assembly.xml
index ff8e0d2fdd..f9daa49e21 100644
--- a/distribution/src/assembly/assembly.xml
+++ b/distribution/src/assembly/assembly.xml
@@ -23,7 +23,7 @@
           xsi:schemaLocation="http://maven.apache.org/ASSEMBLY/2.0.0 http://maven.apache.org/xsd/assembly-2.0.0.xsd">
     <id>bin</id>
     <formats>
-        <format>tar.gz</format>
+        <format>dir</format>
     </formats>
     <fileSets>
         <fileSet>
